### PR TITLE
fix missing translation

### PR DESCRIPTION
**Pull request by carchrae**
a missing translation on a modal.

a minor relabel got committed here too by accident - i would revert, but i think my change is clearer than the original "Notes/Protocol"

lots of small fiddly stuff in my branch like this.  i will persist in committing it.

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -904,6 +904,8 @@ de:
         reload: Zusammenfassung neu laden
         with_extra_charge: 'mit Aufschlag:'
         without_extra_charge: 'ohne Aufschlag:'
+      edit_note:
+        title: Bestellschein bearbeiten
```
Comment (paroga) [inline config/locales/de.yml]: Why did you change it from "Notiz" to "Bestellschein"? IMHO it still refers to "Note"...

Comment (carchrae) [reply config/locales/de.yml]: you have to remember, i don't know german from dutch!  i didn't know what the original was.  :)  

will fix this eve

Comment (paroga) [reply config/locales/de.yml]: AFAIK all untranslated texts are from benni and therefore in German langugage.

File: config/locales/de.yml [hunk 2]
```diff
@@ -904,6 +904,8 @@ de:
         reload: Zusammenfassung neu laden
         with_extra_charge: 'mit Aufschlag:'
         without_extra_charge: 'ohne Aufschlag:'
+      edit_note:
+        title: Notitie bewerken
```
Comment (wvengen) [inline config/locales/de.yml]: This looks like the Dutch translation :) Should be `Notiz bearbeiten`,

Comment (carchrae) [reply config/locales/de.yml]: ha.  ok.  done

**Comment by wvengen** [comment]
I guess we can merge this, unless I'm missing something. @paroga (just to be sure, as you reviewed before).

**Comment by carchrae** [comment]
ha.  well, i can't merge it as i don't have access.  this was perhaps the most work on a PR for the smallest change.  

i would suggest if there are language changes that reviewers just directly commit their language on a PR.  there is also a way to suggest changes in github (it is beta) which would work well in cases like this: https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change
